### ISSUE TITLE
Accessibility - WCAG 1.4.4 Resize text, with IE9 the text is not scalable

### ISSUE THREAD

**Issue by bit-pro-iew-eui** [swiss/styleguide#138]
It should be possible to enlarge either the contents of the entire page or the text alone (with the zoom function of the browser). This is not the case

Original version was ok,

**Comment by ludovicturmel** [comment]
Resize text with IE9 feature will be fix in the next release v.3.0.0
See PR : #444
